csi/moac/volume.ts

```diff
@@ -791,7 +971,12 @@ export class Volume {
   }
 
   // Share replicas as appropriate to allow access from the nexus.
-  async _ensureReplicaShareProtocols(nexusNode: Node, replicaSet: Replica[]) {
+  // It does not throw unless none of the replicas can be shared.
+  // Though replicas that failed to be shared are added to the blacklist.
```

Comment:
Need to update the comment so that we don't refer to "blacklist" if we are no longer using it.